lib/Params.ml

```diff
@@ -432,6 +432,17 @@ let get_cases (c : Conf.t) ~ctx ~first ~last ~cmts_before
   let align_nested_match =
     match (ast.pexp_desc, c.fmt_opts.nested_match.v) with
     | (Pexp_match _ | Pexp_try _), `Align -> last
+    | ( Pexp_extension
+          ( ext
+          , PStr
+              [ { pstr_loc= _
+                ; pstr_desc=
+                    Pstr_eval
+                      ({pexp_desc= Pexp_match _ | Pexp_try _; pexp_loc; _}, _)
+                } ] )
+      , `Align )
+      when Source.extension_using_sugar ~name:ext ~payload:pexp_loc ->
```

Comment:
This function is used quite often, at some point we could consider putting this information in the `Pexp_extension` node.

Reply:
Yes of course, but that turns a simple patch into a complicated project. I don't think my employer would be willing to let me do it.

Reply:
I'm mentioning this in case it spawns ideas. Of course, I want this fix to go in first.